### DISCUSSION TITLE
Live Transcript (Streaming)

### DISCUSSION THREAD

**Discussion by team-deepgram** [deepgram/community#684] [General help] [answered]
Hello, I'm looking to use Deepgram for transcription and we want to do livetranscription, I was wondering where I can find the list of languages that are supported by livetranscription.

Thanks!

> [Posted by rahplays](https://discord.com/channels/1108042150941294664/1226787385535369216/1226787385535369216)

**Comment by jjmaldonis** [comment, marked as answer]
With the exception of hi-Latn, the answer is: streaming is supported for all languages we have here:

https://developers.deepgram.com/docs/models-languages-overview
